helm/reana/README.md

```diff
@@ -103,8 +103,8 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `volume_paths.root_path`                                 | Path to the REANA directory inside the underlying storage volume                     | /var/reana                                      |
 | `volume_paths.shared_volume_path`                        | Path inside the REANA components where the shared volume will be mounted             | /var/reana                                      |
 | `quota.enabled`                                          | Enable user workflow accounting capabilities.                                        | true                                            |
-| `quota.disk_update`                                      | Cronjob pattern representing how often the users disk quota usage should be updated. Leave it empty to deactivate it | "0 3 * * *"     |
-| `quota.termination_update_policy`                        | Resources to calculate quotas on worflow termination. Possible values: "cpu" and "disk". Leave it empty to deactivate workflow termination accounting. | "cpu,disk"                                   |
+| `quota.cronjob_update_policy`                            | Cronjob pattern representing how often the users Disk and CPU quota usage should be updated. Leave it empty to deactivate it | "0 3 * * *" |
```

Comment:
Whilst a name change would be good to do since the nightly updater can do both CPU and Disk, we should beware of changing the names between 0.8.0 and 0.8.1... If we keep this for `maint-0.8`, then we have to keep the names.  If we change the names, then this must go into 0.9 release series only.

(BTW, speaking of changing names, `quota.termination_update_policy` might read a bit ambiguous to some perhaps, as if we were terminating quotas... Using a fuller name such as `quota.workflow_termination_update_policy` would be more understandable, since more explicit. But these kinds of musings would belong only for 0.9 if we want to change both names.)

(Ditto `quota.periodic_update_policy` might read better than `quota.cronjob_update_policy` for similar ambiguity reasons, in order to avoid an impression that we are updating cronjobs, as it were.)

Reply:
Please check/test thoroughly the template (`tpl`) mechanism to be able to be backward compatible among values in `values.yaml`.